src/services/summaryHeatmapService.ts

```diff
@@ -279,3 +284,65 @@ export const getConnectionDetails = (
     ? uniqueKS[Object.keys(uniqueKS)[connectionPage - 1]]
     : ({} as KnowledgeStatement);
 };
+
+export const reorderXAxis = (xAxis: string[]): string[] => {
```

Comment:
question: How does this sorting relates with the json sorting we have implemented [here](https://github.com/MetaCell/sckan-explorer/blob/develop/src/services/hierarchyService.ts#L164), if it relates at all?

Reply:
already discussed on slack